### PR TITLE
Add assets for 'CHARLIE-fcd997'

### PR DESCRIPTION
**Pull request by LuciferArc1**

### CODE REVIEW

**Review by paulmultiversx — APPROVED** [review]

**Comment by paulmultiversx** [comment]
Hi, @LuciferArc1 

Please provide a signature for the latest commit sha: `f13411dd7928a1b25640981de9415008534d96c6` which must be signed with the owner wallet address: 
`erd1ms4lkzsq9gd88he94qsc5qt4a3thdq0a7wxu069zskc7q8ldwckslg2378`

Feel free to use the Sign Message feature from the Utils dApp and paste the signature in a comment: https://utils.multiversx.com/

Thanks.

File: tokens/CHARLIE-fcd997/info.json
Comment (LuciferArc1) [inline tokens/CHARLIE-fcd997/info.json:1]: 0x6b952c9ec60e088686ef778b2ee453aa49eed910ef243f1cfb9ac0bc5a07f060bda2449b72616b6b6f8aaa5b978eb857f0ccd7b5e2916762cb832d613653fa01

**Review by LuciferArc1 — COMMENTED** [review]
0x6b952c9ec60e088686ef778b2ee453aa49eed910ef243f1cfb9ac0bc5a07f060bda2449b72616b6b6f8aaa5b978eb857f0ccd7b5e2916762cb832d613653fa01

**Comment by LuciferArc1** [comment]
0x6b952c9ec60e088686ef778b2ee453aa49eed910ef243f1cfb9ac0bc5a07f060bda2449b72616b6b6f8aaa5b978eb857f0ccd7b5e2916762cb832d613653fa01

**Review by paulmultiversx — APPROVED** [review]

**Pull request merged**